### PR TITLE
doc/tutorial: update for recent code changes

### PR DESCRIPTION
**Pull request by heasterday**
Addresses #1018 and #414

### CODE REVIEW

File: doc/tutorial.rst
Comment (reidpr) [inline doc/tutorial.rst:104]: I'm nervous about including `--force` without explanation, but it's probably good enough for now.

File: doc/tutorial.rst
Comment (reidpr) [inline doc/tutorial.rst:135]: This isn't quite right — most builders can use either name or hash, but Charliecloud uses only name. I'd say this note could be omitted from the tutorial.

File: doc/tutorial.rst
Comment (reidpr) [inline doc/tutorial.rst:143]: I think it would be better to do the tutorial with just one workflow, and support the other with a single sidebar or something saying how to translate.

File: doc/tutorial.rst
Comment (reidpr) [inline doc/tutorial.rst:168]: I wonder if converting images between formats is too much detail for a tutorial? I do like the form we settled on for the workshop with some of the things relegated to "advanced topics" at the end.

File: doc/tutorial.rst
Comment (reidpr) [inline doc/tutorial.rst:192]: Suggest either omitting this or converting it to a real example.

File: doc/tutorial.rst
Comment (reidpr) [inline doc/tutorial.rst:210]: I read recently that "latter" and "former" can be confusing and realized I agree (similarly, a list of things followed by "respectively"). The suggested alternative is to move things closer together so you don't need such words. For example: "Charliecloud can run out of a normal directory or a mounted SquashFS (preferred)." (Though in this case IMO it's the internal SquashFS mount that's preferred, not the manual mount/unmount.)

File: doc/tutorial.rst
Comment (reidpr) [inline doc/tutorial.rst:947]:
```suggestion
   :code:`srun fusermount -u /var/tmp/mpihello-openmpi` 
```

**Review by reidpr — CHANGES_REQUESTED** [review]
OK great work. This is pretty close to a minimum viable product to unblock 0.26. See comments in-line.

File: doc/tutorial.rst
Comment (heasterday) [reply doc/tutorial.rst:53]: How about option #2 with a note stating what it looks like to manually mount/unmount? Do we have documentation on building the correct version of SquashFUSE?

File: doc/tutorial.rst
Comment (heasterday) [reply doc/tutorial.rst:56]: I'll remove

File: doc/tutorial.rst
Comment (heasterday) [reply doc/tutorial.rst:70]: Will do.

File: doc/tutorial.rst
Comment (heasterday) [reply doc/tutorial.rst:168]: I think it's good to be aware that the functionality exists and don't feel that it adds too much complexity. I defer to you though.

File: doc/faq.rst
Comment (heasterday) [reply doc/faq.rst:322]: I will change to `foo`.

File: doc/tutorial.rst
Comment (heasterday) [reply doc/tutorial.rst:210]: Excellent point, I will adjust.

**Review by reidpr — APPROVED** [review]
LGTM, please review my changes and if they look good I'll merge.

File: doc/tutorial.rst
Comment (heasterday) [inline doc/tutorial.rst]: I think it would be good to document this somewhere, if you don't like it here, perhaps a FAQ entry?

Comment (reidpr) [reply doc/tutorial.rst]: Yeah, I agree. FAQ would be good. I was just wanting to keep manual mount/unmount out of the tutorial.

**Comment by heasterday** [comment]
@reidpr This looks good to me

**Pull request merged**
